linux/usage/users.md

```diff
@@ -30,25 +30,21 @@ Upon creating a new user, the contents of `/etc/skel/` will be copied to the new
 
 ## Sudoers
 
-The default `pi` user on Raspbian is a sudoer. This gives the ability to run commands as root when preceded by `sudo`, and to switch to the root user with `sudo su`.
+The default `pi` user on Raspbian is a member of the `sudo` group. This gives the ability to run commands as root when preceded by `sudo`, and to switch to the root user with `sudo su`.
 
-To add a new user to sudoers, type `sudo visudo` (from a sudoer user) and find the line `root    ALL=(ALL:ALL) ALL`, under the commented header `# User privilege specification`. Copy this line and switch from `root` to the username. To allow passwordless root access, change to `NOPASSWD: ALL`. The example below gives the user `bob` passwordless sudo access:
+To add a new user to the `sudo` group, use the adduser command:
 
 ```bash
-# User privilege specification
-root  ALL=(ALL:ALL) ALL
-bob   ALL = NOPASSWD: ALL
+sudo adduser bob sudo
 ```
 
-Save and exit to apply the changes. Be careful, as it's possible to remove your own sudo rights by accident.
-
-You can change the editor the `visudo` command uses (the default is Nano) by entering:
+Note that the user `bob` will be prompted to enter their password when they run `sudo`. This differs from the behaviour of the `pi` user, since `pi` is not prompted for their password. If you wish to remove the password prompt from the new user, create a custom sudoers file and place it in the `/etc/sudoers.d` directory:
 
 ```bash
-update-alternatives --set editor /usr/bin/vim.tiny
+echo 'bob ALL=(ALL) NO PASSWD: ALL' | sudo tee -a /etc/sudoers.d/010_bob-nopasswd
```

Comment:
@andrum99 
Perhaps skip the `-a` option, which is to append to an existing file. Of course with `-a` it works as well to create a new file, but if one accidentally runs the command doubled, or one wants to fix a broken/edited file, this would lead to two/doubled lines. I think the file name is unique enough that there is no real chance to overwrite some existing entry that was wanted to be preserved. So:
```
echo 'bob ALL=(ALL) NO PASSWD: ALL' | sudo tee /etc/sudoers.d/010_bob-nopasswd
```